server/lib/accounts.js

```diff
@@ -91,6 +118,27 @@ Accounts.onCreateUser(function(options, user = {}) {
 		}
 	}
 
+	if (!user.active) {
+		user.emails.some((email) => {
+			const destinations = [];
+
+			RocketChat.models.Roles.findUsersInRole('admin').forEach(function(adminUser) {
+				destinations.push(`${ adminUser.name }<${ adminUser.emails[0].address }>`);
```

Comment:
It's possible that an adminUser doesn't have e-mail on `adminUser.emails[0].address` (for example if the admin only ever logged in using OAuth). You should check if property exists.